realm/realm-jni/build.gradle

```diff
@@ -75,11 +84,24 @@ buildscript {
 
 apply plugin: 'de.undercouch.download'
 
-if (project.clang instanceof String) {
-    ext.clang = Boolean.parseBoolean(project.clang)
+if (ext.debugBuild) {
+    // Debug build should never strip symbols
+    ext.stripSymbols = false
 }
-if (project.forceDownloadCore instanceof String) {
-    ext.forceDownloadCore = Boolean.parseBoolean(project.forceDownloadCore)
+if (ext.coreSourcePath) {
+    // Run the "sh build.sh get-version" to get the core version.
+    new ByteArrayOutputStream().withStream { os ->
```

Comment:
No, it doesn't work... Did you miss anything? `"sh build.sh get-version"` is a string which doesn't have a `execute` method.